index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,900;1,300;1,700&display=swap" rel="stylesheet">  <title>Chopin Concert</title>
+</head>
+<body class="d-flex flex-column" id="home">
+  <header class="menu">
+    <nav class="menu-btn">
+      <div class="menu-btn__burger"></div>
+    </nav>
+    <nav>
+      <section class="menu-d-items" id="dmenu-item">
+        <section class="menu-logo menu-desktop">
+          <a href="index.html"><img src="img\logo.png" alt="logo"></a>
+        </section>
+        <ul class="item">
+          <li class="nav-item-btn"><a href="about.html">About</a></li>
+          <li class="nav-item-btn"><a href="#mainProgram">Program</a></li>
+          <li class="nav-item-btn"><a href="#ticket">Ticket</a></li>
+          <li class="nav-item-btn"><a href="#partnerSection">Partners</a></li>
+          <li class="nav-item-btn"><a href="#performerSection">Performers</a></li>
+        </ul>
+      </section>
+    </nav>
```

Comment:
**OPTIONAL** - To help improve your site's HTML semantics, it is not advisable to have 2 `nav` tags in one page. Please fix this, using CSS you can hide elements you don't want to be displayed. 👍 👍 
**OPTIONAL** - To help improve your site's UI, I suggest you make your nav height the full height of the screen, please fix this 👍 👍 

Your project:
<img width="305" alt="xbzbb" src="https://user-images.githubusercontent.com/73607512/203842609-9207ab09-f6de-4f76-9fe3-6cf6da8e467e.PNG">